stories/carousel/index.stories.tsx

```diff
@@ -115,6 +115,10 @@ export default {
   title: 'Components/Carousel',
   component: Carousel,
   parameters: {
+    /**
+     * There is visual jank when this component loads--this reduces brittleness in Chromatic
```

Comment:
I noticed this when testing but figured it was a fluke, but looking at `radiance-ui.curology.com` it's consistent.